spec/requests/internal/reactions_spec.rb

```diff
@@ -30,12 +30,12 @@
 
     it "returns HTTP Status 200 upon status update" do
       put "/internal/reactions/#{reaction.id}", params: { id: reaction.id, status: "confirmed" }
-      expect(response).to have_http_status(200)
+      expect(response).to have_http_status(:ok)
```

Comment:
Why did you choose to make this change as well as the change on line 38?

Reply:
I didn't :D It was the new `rubocop-rails`, it has [a rule for that](https://github.com/rubocop-hq/rubocop-rails/blob/7ac8eef34b455175ceea4df2291cc0c0ca36defc/config/default.yml#L262). I do agree with the rule though, as we mostly don't use numbers when we use HTTP statuses but their "colloquial" symbolic equivalent :D